pelican/plugins/more_categories/more_categories.py

```diff
@@ -19,13 +19,13 @@ class Category(URLWrapper):
     @property
     def _name(self):
         if self.parent:
-            return self.parent._name + '/' + self.shortname
+            return self.parent._name + "/" + self.shortname
```

Comment:
This is done automatically using the tasks that are present within pelican. In the lint section they are using "black". Black makes it so that single quotes are turned to double quotes to make coding conventions the same across the board. Since black is included in every other plugin and in (latest) pelican, I assume that is what is wanted.

Reply:
While I thought it would be beneficial to have consistent code style across the board, I understand that code style is very much subjective and something that shouldn't be forcibly foisted upon plugin developers who prefer a different style. So plugin authors are free to use their own conventions, as @oulenz has done with More Categories. For example, as you can see, Black is (purposely) not among the tools run during the `lint` task in this repository. So while I fully understand and appreciate your intention here, it would be best to honor this repository's style and not change these quotation marks. :blush:

Reply:
Ok cool. I'll go ahead and undo those then. Sorry about that @oulenz